### DISCUSSION TITLE
Do I need to connect wires?

### DISCUSSION THREAD

**Discussion by wahkawahka** [Opinionated-Blueprints/10-Books-Full-of-Rails#133] [Q&A]
I am working to better understand the setup and use of this blueprint book.  Do I need to connect a red, green or both wires from any of my stations or depots to the red and green wires on the rail blueprints? TIA

**Comment by bwoods89** [comment]
No, you don't need to. Each requester/provider station needs its own local circuit network, so connecting them all to the circuit network on the rail blueprints will actually break your stations.
